### ISSUE TITLE
Vendor / NPC Editor

### ISSUE THREAD

**Issue by bsparks** [ironbane/IronbaneServerLegacy#365]
create a editor to simplify the creation of vendors and other npcs

**Comment by ghost** [comment]
Also including quests? Would be nice to have a simple interface for that as well.

**Comment by bsparks** [comment]
a quest editor will probably be hard, unless we have basic quest templates like "bring x , get y" or "kill x whatevers" currently they are all hand coded

if we can template them out into stuff that you can just plug data into then maybe

**Comment by kojagiri1934** [comment]
Monsters == NPC? So maybe #364 is a duplicate

**Comment by bsparks** [comment]
well there are a lot of complex rules for Vendors vs Mobs so I think having them separate might be good. don't want to have a complex UI

**Comment by kojagiri1934** [comment]
Agreed re. complexity; however would suggest having one editor and a module that provides for extra. To avoid double work.

**Comment by bsparks** [comment]
maybe you're right, we'll see when I get into it what makes sense
